example/lib/test_scenarios/peripheral_test_operations.dart

```diff
@@ -67,14 +67,33 @@ class PeripheralTestOperations {
     log("negotiated MTU $negotiatedMtu");
   }
 
+  Future<void> readCharacteristicForPeripheral() async {
+    log("Reading temperature");
+    CharacteristicWithValue readValue = await peripheral.readCharacteristic(
+        SensorTagTemperatureUuids.temperatureService,
+        SensorTagTemperatureUuids.temperatureDataCharacteristic);
+    log("Read temperature value ${_convertToTemperature(readValue.value)}C");
+  }
+
+  Future<void> writeCharacteristicForPeripheral() async {
+    log("Write to characteristic via peripheral");
+    await peripheral.writeCharacteristic(
+        SensorTagTemperatureUuids.temperatureService,
+        SensorTagTemperatureUuids.temperatureDataCharacteristic,
+        Uint8List.fromList([0, 200, 40, 0]),
```

Comment:
1. why did you changes `Uint8List.fromList([1]),` to `Uint8List.fromList([0, 200, 40, 0]),`?
2. Why it is not introduced into the example on develop?

Reply:
Actually it was changed having only simulator example in mind. It would throw an error as data characteristic is not writable in real sensor tag. 
For now I will just leave writing "1" to characteristic config, but fully working example for characteristics I/O operations will be introduced in characteristics monitoring PR.